test/fixtures/output/organization/list.golden

```diff
@@ -0,0 +1,5 @@
+  Current |   ID    | Name  
```

Comment:
Should we be sorting this list? What do we do for `environment list` or `kafka cluster list`?

Reply:
`kafka cluster list` appears to do some type of sorting (I didn't investigate too much, but w/ 4 basic clusters in the same cloud and region it sorts by name).

Both `environment list` and `organization list` are sorted by creation date in the response (can be checked by looking at `GetMetadata()`), and after that it's just whatever sorting happens in `Print()`. Which, at least for `environment` and `organization`, looks to be a reversal of the response order but with the active resource always listed last.

Reply:
For the integration tests here, I think the reason it looks unsorted is because of that last bit where the active one always gets moved to the bottom of the list.

Reply:
Ah, I think it sorts by default, but we should be ignoring the first column